README.md

```diff
@@ -12,6 +12,109 @@ OpenSearch Ruby Client
 
 **opensearch-ruby** is [a community-driven, open source fork](https://aws.amazon.com/blogs/opensource/introducing-opensearch/) of elasticsearch-ruby licensed under the [Apache v2.0 License](LICENSE.txt).
 For more information, see [opensearch.org](https://opensearch.org/).
+
+## Setup
+
+To add the client to your project, install it using [RubyGem](https://rubygems.org/)
+
+`gem install opensearch`
```

Comment:
or install it from a source code checkout:

```
git clone https://github.com/elasticsearch/opensearch-ruby.git
cd opensearch-ruby/opensearch
gem build opensearch.gemspec
gem install opensearch-<version>.gem
```

Reply:
Done